### PR TITLE
Add ability to select who to send reminder emails to

### PR DESCRIPTION
**Pull request by wdoug**
This is a big hack, but it adds value, and it was requested to be done before they send out the next reminder emails tomorrow morning.

### CODE REVIEW

File: lib/components/admin/Admin.js [hunk 1]
```diff
@@ -44,6 +44,24 @@ export default class Admin extends React.Component {
     props.loadDonations({from: donationsFrom, to: donationsTo});
   }
 
+  _sendReminderEmails() {
+    const locationIds = this.refs['coordination-table'].getSelectedLocationIds();
+    const locationAddresses = locationIds.map(id => {
+      const {address, city, state, zipcode} = this.props.admin.locations.find(location => +location.id === +id);
```
Comment (dviramontes) [inline lib/components/admin/Admin.js:50]: nice use of `.find()` :clap:

**Comment by dviramontes** [comment]
:+1:

File: lib/components/admin/Admin.js (hunk 1 above)
Comment (wdoug) [inline lib/components/admin/Admin.js:50]: Thanks. Like my silly parseFloat trick?

Comment (dviramontes) [inline lib/components/admin/Admin.js:50]: yeah its pretty rad :)

**Pull request merged**